### PR TITLE
Update social icons

### PR DESCRIPTION
**Pull request by mdo**
Pulling this into the main repo for v4 and v5 first instead of starting in the icons repos. Wondering if we need to replace the social image as well these icons.

/cc @XhmikosR 

TODO:

- [ ] update favicon.ico
- [x] fix filenames
- [ ] update OG image
- [x] remove unused images
- [x] compress images
- [ ] make sure Windows favicons look good
- [ ] make sure Twitter summary is what we want now

### CODE REVIEW

**Comment by XhmikosR** [comment]
I'm gonna tweak this tomorrow and also compress them, but I'll keep the 8-bit patch as a separate one so don't squash it please.

And yeah, we should also redo the OG images. BTW I'm gonna rename some of the images

**Comment by XhmikosR** [comment]
Also, are you sure we should backport this to v4?

File: site/layouts/partials/social.html [hunk 1]
```diff
@@ -1,18 +1,16 @@
-{{ "<!-- Twitter -->" | safeHTML }}
 <meta name="twitter:card" content="{{ if .IsHome }}summary_large_image{{ else }}summary{{ end }}">
 <meta name="twitter:site" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:creator" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:title" content="{{ .Title | markdownify }}">
 <meta name="twitter:description" content="{{ .Page.Params.description | default .Site.Params.description | markdownify }}">
-<meta name="twitter:image" content="{{ if .IsHome }}{{ .Site.Params.social_logo_path | absURL }}{{ else }}{{ .Site.Params.social_image_path | absURL }}{{ end }}">
+<meta name="twitter:image" content="{{ .Site.Params.social_image_path | absURL }}">
```
Comment (XhmikosR) [inline site/layouts/partials/social.html]: Just so that we are on the same page, you mean to switch to one OG image?

Comment (mdo) [reply site/layouts/partials/social.html]: I think we're fine to have a single one? Open to feedback though!

**Comment by mdo** [comment]
Sidenote: I use https://metatags.io to preview how these OG cards look. Holler at me if there's another tool that's similar or better.

**Comment by mdo** [comment]
> Also, are you sure we should backport this to v4?

If the team doesn't want to, I'm fine taking it to v5 only.

**Comment by XhmikosR** [comment]
@mdo we need the site/static/docs/4.3/assets/brand SVGs to be updated with the ones you used.

Then I will generate a new favicon.ico too.

File: site/static/docs/4.3/assets/img/favicons/safari-pinned-tab.svg [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none" viewBox="0 0 192 192">
+  <path fill="#000" d="M0 0h192v192H0z"/>
+  <path fill="#fff" fill-rule="evenodd" d="M136 20H56c-19.882 0-36 16.118-36 36v80c0 19.882 16.118 36 36 36h80c19.882 0 36-16.118 36-36V56c0-19.882-16.118-36-36-36zm-80-4c-22.091 0-40 17.909-40 40v80c0 22.091 17.909 40 40 40h80c22.091 0 40-17.909 40-40V56c0-22.091-17.909-40-40-40H56z" clip-rule="evenodd"/>
+  <path fill="#fff" fill-rule="evenodd" d="M101.367 136H66.621V51.45h33.984c15.879 0 25.43 8.085 25.43 21.093 0 8.848-6.504 16.758-14.824 18.164v.996c11.425 1.172 19.043 9.317 19.043 20.332 0 14.883-10.84 23.965-28.887 23.965zM79.746 62.582V87.25h15.586c11.601 0 17.636-4.277 17.636-12.305 0-7.793-5.683-12.363-15.41-12.363H79.746zm18.984 62.285H79.746V97.621h18.222c12.364 0 18.868 4.629 18.868 13.477 0 8.964-6.27 13.769-18.106 13.769z" clip-rule="evenodd"/>
```
Comment (midzer) [inline site/static/docs/4.3/assets/img/favicons/safari-pinned-tab.svg]: Just tested running the .svg through https://jakearchibald.github.io/svgomg/ with `Precision: 2` we can save about 20% file size with no (visual) loss in detail

**Review by midzer — CHANGES_REQUESTED** [review]
Just a minor file size improvement for iOS `.svg` possible.

And I double checked `android-chrome-512x512.png` for possible size reduction via `optipng`. Result: it is already optimized :+1:

File: site/static/docs/4.3/assets/img/favicons/safari-pinned-tab.svg (hunk 2 above)
Comment (XhmikosR) [reply site/static/docs/4.3/assets/img/favicons/safari-pinned-tab.svg]: Thanks, but I'm not interested in further SVG optimizations

**Comment by mdo** [comment]
Closing as stale, and as tackled in #31042.

**Pull request closed without merging**